web/app/scripts/views/record/add-edit-partial.html

```diff
@@ -42,7 +42,7 @@
                             </div>
                         </div>
                         <div class="col-md-12">
-                            <div class="map" leaflet-map driver-embed-map style="height:300px;"></div>
+                            <div class="map" leaflet-map driver-embed-map editable="true"></div>
```

Comment:
Yeah, I was mostly thinking of it from the perspective of the embedded map I'm adding on the details view. There I already have the latitude and longitude on the controller at the time this directive is rendered, so it'd be great to be able to just pass them in.